### PR TITLE
WIP: 2018 Constitution Revision

### PR DESCRIPTION
**Pull request by azharichenko**
## Why?
As fresh leadership with new eyes on the club, we really wanted to define our purpose, roles, etc. so we can be a leading example of what is club is for the future and have clear direction how to run this club this year.

## Goals
- [x] Have a well-defined purpose
- [x] Have our three pillars mentioned in our purpose defined
  - [x] Define our pillar of the community
  - [x] Define our pillar of education
  - [x] Define our pillar of professional development
---
- [x] Have core executive board roles defined
- [x] Make current positions be known as executive board and have officer as a generic role we all fulfill
- [x] Have officer role defined
- [x] Remove the idea of committees from the constitution
---
- [x] Added section on initiatives 
---
- [x] Define Community Partner
- [x] Define Sponsor
---
- [x] Define what it is to be a member
- [x] Define our stance on data (what we collect, what we use it for, what we disclose to sponsor, etc.)

### CODE REVIEW

File: Constitution.md [hunk 1]
```diff
@@ -145,7 +152,15 @@ The advisor is not required to provide any particular services to the Pitt CSC,
 
 ## XXIII) External Affiliations
 
-The Pitt CSC is not affiliated with any external organizations.
+The Pitt CSC has two forms of external affiliation: Sponsors and Community Partners.
```
Comment (varughese) [inline Constitution.md]: I think adding this is pretty important. We need to define the difference between these anyways. The difference is pretty much sponsors we get money and advertise them, community partners we partner with and cohost events. Code & Supply would be a community partner, Microsoft would be a sponsor

File: Constitution.md [hunk 2]
```diff
@@ -135,23 +138,46 @@ Vacancies will be filled by appointment from the board.  The replacement will ho
...
+
+### What we collect?
+
+Our collection is limited to attendence of meetings, activity in our group chat (only the time when a message is sent but not the contents of that message), activity on our Github organization (including commits and the impact of those changes), or details about a member from setting up with our attendance system, surveys, check-in or sign-up sheets, events (such as resume review) and our newsletter. TODO: Videos?, Attendance, Instragram?
+
+### What is the data used for?
+
+The information is used to evaluate various things such as how effective a particular events is, how attendence changes over the time of a semester, etc. Activity of our members helps the officers figure whether an indivdual is a member of the club, how engage they are with a project, and in special cases which members should be given a high prority for certain events (such as field trips).
```
Comment (varughese) [inline Constitution.md]: attendance

File: Constitution.md [hunk 3]
```diff
@@ -135,23 +138,46 @@ Vacancies will be filled by appointment from the board.  The replacement will ho
...
+
+### What data is given out?
+No data on a particular member, except for any information that a member has consented for the officers to give out, will be givin to sponsors. 
+
+The club gives out general statistics about 
+
```
Comment (varughese) [inline Constitution.md]: resume book by consent of the members

**Comment by azharichenko** [comment]
@haydenstec Can you please revise the section about data, I think I got a good bit of I wanted to get out but you have been pretty good at writing stuff on the constitution and wanted to see if you could make it more professional and formalized.

File: Constitution.md [hunk 4]
```diff
@@ -145,7 +152,15 @@ The advisor is not required to provide any particular services to the Pitt CSC,
...
-The Pitt CSC is not affiliated with any external organizations.
+The Pitt CSC has two forms of external affiliation: Sponsors and Community Partners.
+
+### Sponsor
+
+Pitt CSC Sponsors support Pitt CSC and its activities financially. Sponsors may be invited to events and programs, networking opportunities, limited access to Pitt CSC data, and advertisement and exposure within the club.
+
+### Community Partner
+
+Pitt CSC Community Partners are reciprocal relationships with organizations based in the Greater Pittsburgh Area. Pitt CSC recognizes Community Partners for continued aid in organizing cohosted programs, encouraging interactions between organization members, and...
```
Comment (rohit-ganguly) [inline Constitution.md]: We should also try and say that the Community Partner initiative is to help do our part to the software community in Pittsburgh as a whole.

**Pull request merged**